introspection-engine/connectors/sql-introspection-connector/src/introspection_helpers.rs

```diff
@@ -149,21 +149,35 @@ pub(crate) fn calculate_scalar_field(
     } else {
         calculate_scalar_field_type(column, family)
     };
-
-    let (is_commented_out, documentation) = match field_type {
-        FieldType::Unsupported(_) => (true, Some("This type is currently not supported.".to_string())),
-        _ => (false, None),
-    };
-
     let arity = match column.tpe.arity {
         _ if column.auto_increment && field_type == FieldType::Base(ScalarType::Int, None) => FieldArity::Required,
         ColumnArity::Required => FieldArity::Required,
         ColumnArity::Nullable => FieldArity::Optional,
         ColumnArity::List => FieldArity::List,
     };
 
+    let (default_value, dbgenerated_string) = calculate_default(table, &column, &arity);
+
+    let (is_commented_out, documentation) = match (&field_type, dbgenerated_string) {
+        (FieldType::Unsupported(_), None) => (true, Some("This type is currently not supported.".to_string())),
+        (FieldType::Unsupported(_), Some(default)) => (
+            true,
+            Some(
+                format!(
+                    "This type is currently not supported.\nThis field's default value can currently not be parsed: `{}`.",
+                    default
+                )
+                .to_string(),
```

Comment:
nitpick: the `.to_string(),` is unnecessary, can be removed

Reply:
Done